scripts/generateThorLongtailTokens/generateThorLongtailTokens.ts

```diff
@@ -0,0 +1,82 @@
+import type { ChainReference } from '@shapeshiftoss/caip'
+import { ASSET_NAMESPACE, CHAIN_NAMESPACE, toAssetId } from '@shapeshiftoss/caip'
+import axios from 'axios'
+import fs from 'fs'
+import path from 'path'
+
+type Token = {
+  chainId: number
+  address: string
+  decimals: number
+  name: string
+  symbol: string
+  logoURI: string
+}
+
+type TokenList = {
+  name: string
+  logoURI: string
+  keywords: string[]
+  version: {
+    major: number
+    minor: number
+    patch: number
+  }
+  tokens: Token[]
+}
+
+const ethUrl =
+  'https://gitlab.com/thorchain/thornode/-/raw/4601fe74d5f990052e11d29b036dce5b727da990/common/tokenlist/ethtokens/eth_mainnet_latest.json'
+const avaxUrl =
+  'https://gitlab.com/thorchain/thornode/-/raw/4601fe74d5f990052e11d29b036dce5b727da990/common/tokenlist/avaxtokens/avax_mainnet_latest.json'
+const bscUrl =
+  'https://gitlab.com/thorchain/thornode/-/raw/4601fe74d5f990052e11d29b036dce5b727da990/common/tokenlist/bsctokens/bsc_mainnet_latest.json'
```

Comment:
do we want this pinned to that commit hash of have it track master?

Reply:
i guess it can disappear with refactors their side if we track master

Reply:
on second thought we should track master and make this script fail on 404 so we are always using latest, with prompt to update the urls as required

Reply:
Indeed, switched to their `develop` branch and added error logging for each URL in case something goes wrong in https://github.com/shapeshift/web/pull/5638/commits/f934129d2ff4c299ba286c9549ba9e24a96fdb5c.